source/restructuredtext.rst_

```diff
@@ -0,0 +1,258 @@
+段落
+-----
+
+段落是由一或多个空白行分隔的文本块。
+
+同一段落的所有行都必须
+左对齐到相同的缩进级别。
+
+
+标题
+-----
+
+在标题下加标点符号便可以创建标题，
+标点符号至少要和标题一样长，如上标题。
+
+子标题
+^^^^^^^
+
+子标题下的段落。
+
+子子标题
+"""""""""
+
+有些编辑器显示的标题和标点符号长度可能不准确，
+不确定的时候可以使用 vim 编辑器检查。
+
+标题级别与使用的标点符号无关，
+建议编写文档时使用统一的标点符号来确定标题级别。
+
+
+
+
+
+
+格式化文字
+----------
+
+使用 **粗体** 标记
+
+
+使用 *斜体* 标记
+
+
+使用 ``wc /etc/passwd`` 查看密码设置
+
+
+链接
+-----
+
+
+外部链接
+^^^^^^^^
+
+seismo-learn@163.com
+
+https://seismo-learn.org/seismology101
+
+
+`地震“学”主站 <https://seismo-learn.org/>`__
+
+
+内部链接
+^^^^^^^^^
+
+链接文档位于同一目录： :doc:`general-guidelines`
+
+
+使用绝对路径（source 目录是根目录）： :doc:`/general-guidelines`
+
+
+同文档内的标题： `reStructuredText 备忘单`_ 、`格式化文字`_
+
+
+列表
+-----
+
+**无序列表：**
+
+- 地震“学”软件
+- 地震“学”科研
+  入门教程
+- 地震“学”参考书
+
+
+**有序列表：**
+
+1. 地震“学”软件
+2. 地震“学”科研入门教程
+3. 地震“学”
+   参考书
+
+
+
+
+**术语定义列表：**
+
+地震“学”
+    地震“学”是一群地震学的搬砖工成立的团队。
+
+    核心目标是提供地震学科研入门培训材料、
+    整理和维护地震学学习资料、
+    促进地震学中文社区的交流和发展。
+
+地震“学”科研入门教程
+    地震“学”小组撰写的《地震“学”科研入门教程》
+    主要目的是帮助地震学新手快速入门，
+    以尽快开展实际的科研工作。
+
+
+
+
+
+代码
+-----
+
+使用 echo 命令输入字符：
+
+::
+
+    # 使用 echo（回声）命令输出 Hello World!
+    $ echo "Hello World"
+
+
+
+使用 echo 命令输入字符::
+
+    # 使用 echo（回声）命令输出 Hello World!
+    $ echo "Hello World"
+
+
+
+使用 echo 命令输入字符：
+
+.. code-block:: bash
+
+   # 使用 echo（回声）命令输出 Hello World!
+   $ echo "Hello World"
+
+
+表格
+-----
+
+.. table:: 近几年的 macOS 系统版本号（简单表格）
+   :align: center
+
+   ============= ============= ==============
+   版本号         代号           发布日期
+   ============= ============= ==============
+   macOS 10.13   High Sierra   2017 年 9 月
+   macOS 10.14   Mojave        2018 年 9 月
+   macOS 10.15   Catalina      2019 年 10 月
+   macOS 11      Big Sur       2020 年 9 月
+   ============= ============= ==============
+
+
+
+.. table:: 近几年的 macOS 系统版本号（网格表格）
+   :align: center
+
+   +-------------+-------------+---------------+
+   | Version     | Codename    | Release Date  |
+   +=============+=============+===============+
+   | macOS 10.13 | High Sierra | 2017/09       |
+   +-------------+-------------+---------------+
+   | macOS 10.14 | Mojave      | 2018/09       |
+   +-------------+-------------+---------------+
+   | macOS 10.15 | Catalina    | 2019/10       |
+   +-------------+-------------+---------------+
+   | macOS 11    | Big Sur     | 2020/09       |
+   +-------------+-------------+---------------+
```

Comment:
We should avoid using this complicated grid table.

Reply:
I agree. Too complicate.

Reply:
Remove grid table in 2aa1f9f.